enhancements/proposed/20201230-containerd-default/container-default.md

```diff
@@ -0,0 +1,41 @@
+# CRI: Containerd by default
+
+* First proposed: 2020-11-08
+* Authors: Anders F Björklund (@afbjorklund)
+
+## Reviewer Priorities
+
+Please review this proposal with the following priorities:
+
+*   Does this fit with minikube's [principles](https://minikube.sigs.k8s.io/docs/concepts/principles/)?
+*   Are there other approaches to consider?
+*   Could the implementation be made simpler?
+*   Are there usability, reliability, or technical debt concerns?
+
+Please leave the above text in your proposal as instructions to the reader.
+
+## Summary
+
+Change default container runtime, from current "docker" to replacement "containerd".
+
+## Goals
+
+* Change from docker to containerd as default
+* Still allow building images with minikube
+
+## Non-Goals
+
+* Remove the docker support from minikube
+* Change anything in the docker driver
+
+## Design Details
+
+The containerd container runtime is already included, and is passing certification.
+
+Unlike Docker, will need to include the CRI (runtime) and CNI (network) by default.
+
+Use BuildKit as a complement to Containerd, for producing an image from Dockerfile.
```

Comment:
add link to BuildKit to clarify which one you mean.

Reply:
Ok